packages/workbox-background-sync/src/lib/request-queue.js

```diff
@@ -96,6 +96,7 @@ class RequestQueue {
    * @private
    */
   async push({request}) {
+    await this._initializationPromise;
```

Comment:
With an async method like `_getQueue()` you'd never reference `this._queue` and always use the `_getQueue` method, this means that no one can trip up like has been done with `saveQueue()`.